src/Compat/CompatController.php

```diff
@@ -386,8 +386,11 @@ public function sendMultipartUpdate(BaseHtmlElement ...$additionalControls)
      * Instruct the client to side-load additional updates
      *
      * If an item in the given array is indexed by an integer, its value will be used by the client to refresh
-     * the parent of the element identified by it. If indexed by a string, the client will use this index to
-     * identify a container (by id) and will use the value (a URL) to load content into it.
+     * the parent of the element identified by it. The values are expected to be a valid CSS selectors such
+     * as `.foo`, `#foo`. If indexed by a string, the client will use this index to identify a container (by id) and
+     * will use the value (a URL) to load content into it. Since Icinga Web >= 2.12, the indices can be specified with
+     * or without the `#` indicator. However, when you're using older Icinga Web versions, you have to specify the
```

Comment:
~~However, when you're using older Icinga Web versions~~
If you require compatibility with older Icinga Web versions,